dashboard/db/migrate/20180620110434_create_pd_international_optins.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePdInternationalOptins < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_international_optins do |t|
+      t.references :user, index: true
+      t.text :form_data
```

Comment:
Is user required?
Presumably form_data is required, and should be not null.

Also please move the migration out into its own PR (per convention, in case we have to roll back).